minidump-processor/src/process_state.rs

```diff
@@ -917,8 +917,8 @@ Unknown streams encountered:
                 "cpu_arch": sys.cpu.to_string(),
                 "cpu_info": sys.cpu_info,
                 "cpu_count": sys.cpu_count,
-                // optional
-                "cpu_microcode_version": sys.cpu_microcode_version,
+                // optional, print as hex string
+                "cpu_microcode_version": sys.cpu_microcode_version.map(|num| format!("{:#018x}", num)),
```

Comment:
Traditionally we haven't padded this value and other places where it's used don't pad it either (think `/proc/cpuinfo` or Intel's documentation) so I'd leave it unpadded for consistency.